src/test/scala/com/microsoft/hyperspace/SampleData.scala

```diff
@@ -31,7 +31,7 @@ object SampleData {
     ("2018-09-03", "576ed96b0d5340aa98a47de15c9f87ce", "facebook", 2, 3000),
     ("2018-09-03", "50d690516ca641438166049a6303650c", "ibraco", 2, 1000),
     ("2019-10-03", "380786e6495d4cd8a5dd4cc8d3d12917", "facebook", 2, 3000),
-    ("2019-10-03", "ff60e4838b92421eafc3e6ee59a9e9f1", "mi perro", 2, 2000),
+    ("2019-10-03", "ff60e4838b92421eafc3e6ee59a9e9f1", "miperro", 2, 2000),
```

Comment:
Could you create an issue to tracking this problem - spaces in the filepath?

Reply:
#245